game_template does not build on macOS

```sh
$ flutter run -d macos
Running "flutter pub get" in game_template...                    1,678ms
Launching lib/main.dart on macOS in debug mode...
Running pod install...                                              4.1s
/Users/craiglabenz/.pub-cache/hosted/pub.dartlang.org/games_services-2.0.8/macos/Classes/SwiftGamesServicesPlugin.swift:67:25: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardEntry
    let reportedScore = GKScore(leaderboardIdentifier: leaderboardID)
                        ^
/Users/craiglabenz/.pub-cache/hosted/pub.dartlang.org/games_services-2.0.8/macos/Classes/SwiftGamesServicesPlugin.swift:69:5: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardEntry
    GKScore.report([reportedScore]) { (error) in
    ^
/Users/craiglabenz/.pub-cache/hosted/pub.dartlang.org/firebase_crashlytics-2.7.2/macos/Classes/FLTFirebaseCrashlyticsPlugin.m:7:9: fatal error: 'ExceptionModel_Platform.h' file not found
#import "ExceptionModel_Platform.h"
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
** BUILD FAILED **

Building macOS application...
Exception: Build process failed
```

Its web build does work.

--- a/game_template/pubspec.yaml
+++ b/game_template/pubspec.yaml
@@ -24,7 +24,7 @@ dependencies:
   # delete the relevant line below, and get rid of any Dart code
   # that references the dependency.
   firebase_core: ^1.15.0  # Needed for Crashlytics below
-  firebase_crashlytics: ^2.6.3  # Error reporting
+  firebase_crashlytics: ^2.8.1  # Error reporting
   games_services: ^2.0.7  # Achievements and leaderboards
   google_mobile_ads: ^1.1.0  # Ads
   in_app_purchase: ^3.0.1  # In-app purchases

